src/link.rs

```diff
@@ -264,14 +265,9 @@ impl OpenGraphTag {
 
 impl From<PathBuf> for Link {
     fn from(path: PathBuf) -> Self {
-        let file = File::open(path).expect("Open link file");
-        let mut zip = zip::ZipArchive::new(file.try_clone().unwrap())
-            .expect("Open zip archive");
-        let j_raw = zip
-            .by_name("link.json")
-            .expect("Find link.json in the zip archive");
-
-        let j = serde_json::from_reader(j_raw).expect("Parse link.json");
+        let j_raw = std::fs::read(path).expect("Open link file");
+        let j =
+            serde_json::from_slice(j_raw.as_slice()).expect("Parse link.json");
```

Comment:
Error message is misleading now, am I right better phrase it as "Parse .link file failed"?